src/controllers/survey.js

```diff
@@ -0,0 +1,56 @@
+const survey = require('../models/survey');
+const { updateDailySurveys } = require('./dailySurveys');
+// Provice volunteersId and get all matching surveys
+async function getSurveys(req, res) {
+  const id = req.body.volunteersId;
+  try {
+    const allSurveys = await survey.findAll({
+      where: {
+        volunteerId: id,
+      },
+    });
+    res.status(200).json(allSurveys);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error('getSurveys failed:', err);
+    res.sendStatus(500);
+  }
+}
+
+// Recieve a new survey, save it both as a single instance and update the value of dailySurveys
+async function saveSurvey(req, res) {
+  const newSurvey = req.body.survey;
+  const { id } = req.body.volunteersId;
+  try {
+    survey.create({
+      location: newSurvey.location,
+      fish_status: newSurvey.fish_status,
+      fish_species: newSurvey.fish_species,
+      fish_count: newSurvey.fish_count,
+      image_url: newSurvey.image_url,
+      comments: newSurvey.comments,
+      volunteersId: id,
+    });
+    // update the value in dailySurveys
+    const isSaved = updateDailySurveys(
```

Comment:
I am not sure we should be inserting data in two places. We insert in one table and aggregate later. Not really sure what this is for. Let's have a heads up meeting to look at the schema together?